lib/rubocop/rspec/description_extractor.rb

```diff
@@ -61,6 +65,14 @@ def documented_constant
           yardoc.to_s
         end
 
+        def inherits_from_cop?
+          # YARD superclass resolution is a bit flaky: All classes loaded before
+          # RuboCop::Cop::WorkaroundCop are shown as having RuboCop::Cop as
+          # superclass, while all the following classes are listed as having
+          # RuboCop::Cop::RSpec::Cop as their superclass.
+          COP_CLASS_NAMES.include?(yardoc.superclass.path)
```

Comment:
Alternatively, `yardoc.superclass.path.include?('RuboCop::Cop')` will work for anything under the RuboCop::Cop namespace

Reply:
It would, but by coincidence. I prefer this more explicit working around the Yard bug.